modules/monitoring-support-considerations.adoc

```diff
@@ -0,0 +1,24 @@
+:_module-type: CONCEPT
+// Module included in the following assemblies:
+//
+// * assemblies/configuring-the-monitoring-stack.adoc
+
+[id="support-considerations_{context}"]
+= Support considerations for monitoring
+
+[role="_abstract"]
+The following modifications are explicitly not supported:
+
+* *Creating additional `ServiceMonitor`, `PodMonitor`, and `PrometheusRule` objects in the `openshift-&#42;`, and `kube-&#42;` projects.*
+* *Modifying any resources or objects deployed in the `openshift-monitoring` or `openshift-user-workload-monitoring` projects.* The resources created by the {product-title} monitoring stack are not meant to be used by any other resources, as there are no guarantees about their backward compatibility.
```

Comment:
The configmap in openshift-user-workload-monitoring is intended to be managed by dedicated-admins

Reply:
I removed that part of the sentence but kept the rest.